packages/wrangler/src/index.tsx

```diff
@@ -501,6 +511,48 @@ export function createCLIParser(argv: string[]) {
 		}
 	);
 
+	wrangler.command(
+		"deployments",
+		"🚢 Logs the 10 most recent deployments with 'Version ID', 'Version number','Author name', and 'Latest deploy'",
+
+		(yargs) => {
+			yargs.option("name", {
+				describe: "The name of your worker",
+				type: "string",
+			});
+		},
+		async (deploymentsYargs: ArgumentsCamelCase<{ name: string }>) => {
+			await printWranglerBanner();
+			const config = readConfig(
+				deploymentsYargs.config as ConfigPath,
+				deploymentsYargs
+			);
+			const accountId = await requireAuth(config);
+			const scriptName = getScriptName(
+				{ name: deploymentsYargs.name, env: undefined },
+				config
+			);
+			const scriptMetadata = await fetchResult<ServiceMetadataRes>(
+				`/accounts/${accountId}/workers/services/${scriptName}`
+			);
+
+			const scriptTag = scriptMetadata.default_environment.script.tag;
+			const deployments = await fetchResult<DeploymentListRes>(
+				`/accounts/${accountId}/workers/versions/by-script/${scriptTag}`
+			);
+
+			const versionMessages = deployments.versions.map(
+				(versions, index) =>
+					`\nVersion ID: ${versions.version_id}\nVersion number: ${
+						versions.version_number
+					}\nAuthor name: ${versions.metadata.author_name}\nLatest deploy: ${
```

Comment:
We should use the email in this line instead:
```
s/name/email/g
```

Reply:
Ok I can do that.

Reply:
Wait is there a property called email yet? 
Currently this is what I am expecting: 
```ts
export type DeploymentListRes = {
	versions: {
		version_id: string;
		version_number: string;
		metadata: {
			author_id: string;
			author_name: string;
			source: "api" | "dash" | "wrangler" | "terraform" | "other";
			created_on: string;
			modified_on: string;
		};
		preview: {
			active: boolean;
			url: string;
		};
		resources: {
			script: string;
			bindings: unknown[];
		};
	}[];
};
```

Reply:
Looks you already did some changes, make sure to change all instances that refer to author name to author email (in readme(s), comments, etc).